src/solidClientHelpers/policies.js

```diff
@@ -47,7 +47,7 @@ export function getResourcePoliciesContainerPath(
   return getContainerUrl(getPolicyUrl(resource, policiesContainerUrl));
 }
 
-export function getNamedPolicyResourceUrl(
+export function getPolicyResourceUrl(
```

Comment:
It sounds like maybe we want to create a Policy model, and move some of these functions into `src/models/policy`. Not strictly necessary, so I'll leave you to decide whether to do it.

Reply:
Moved them. The model is now essentially a duplicate of the old helpers file, but it should be useful when getting rid of those helpers.